python_modules/dagster/dagster/core/definitions/repository_definition.py

```diff
@@ -631,17 +633,20 @@ def from_list(cls, repository_definitions):
 
             elif isinstance(definition, AssetGroup):
                 asset_group = definition
+
                 if asset_group.all_assets_job_name() in pipelines_or_jobs:
                     raise DagsterInvalidDefinitionError(
                         "When constructing repository, attempted to pass multiple AssetGroups. There can only be one AssetGroup per repository."
                     )
-                pipelines_or_jobs[asset_group.all_assets_job_name()] = build_assets_job(
-                    asset_group.all_assets_job_name(),
-                    assets=asset_group.assets,
-                    source_assets=asset_group.source_assets,
-                    resource_defs=asset_group.resource_defs,
-                    executor_def=asset_group.executor_def,
-                )
+                with warnings.catch_warnings():
+                    warnings.simplefilter("ignore", category=ExperimentalWarning)
+                    pipelines_or_jobs[asset_group.all_assets_job_name] = build_assets_job(
```

Comment:
I think some parentheses are missing here, maybe because of a merge conflict

Reply:
good spot - this file got destroyed by a merge conflict